config.toml

```diff
@@ -45,11 +45,6 @@ link = "https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.5/flowbite.min.css"
 link = "https://unpkg.com/@highlightjs/cdn-assets/styles/default.min.css"
 
 ## JavaScript Plugins
-[[params.plugins.js]]
```

Comment:
The Font Awesome CDN is listed in this file as well. It replaces the kit. I also checked the site to make sure icons still work after removing the kit.